<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tablaClientes</class>
 <widget class="QWidget" name="tablaClientes">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>589</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1245</width>
    <height>589</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1245</width>
    <height>589</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Clientes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Recursos/img/bg.qrc">
    <normaloff>:/prefijoNuevo/brianda1.png</normaloff>:/prefijoNuevo/brianda1.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2C3E50; /* Fondo oscuro */

}
</string>
  </property>
  <widget class="QFrame" name="frameDeBotones">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(38, 45, 67, 100);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelId">
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LabelNombre">
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_nombre"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="LabelApell">
       <property name="text">
        <string>Apellidos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_apellidos"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LabelNumId">
       <property name="text">
        <string>Num_Identificación</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_numId"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelFecha">
       <property name="text">
        <string>Fecha_Nacimiento</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelPas">
       <property name="text">
        <string>Pais</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_pais"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelTLF">
       <property name="text">
        <string>Teléfono</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_telefon"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelMail">
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_mail"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelSexo">
       <property name="text">
        <string>Sexo</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_sexo"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelMenores">
       <property name="text">
        <string>Menores</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineEdit_menores"/>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="date_fechaNac">
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="botonLeer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Leer</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #2C3E50</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Recursos/img/bg.qrc">
         <normaloff>:/prefijoNuevo/magnet.svg</normaloff>:/prefijoNuevo/magnet.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="botonCrear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Crear</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #2C3E50</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Recursos/img/bg.qrc">
         <normaloff>:/prefijoNuevo/new.svg</normaloff>:/prefijoNuevo/new.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="BotonUpdate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Actualizar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #2C3E50</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Recursos/img/bg.qrc">
         <normaloff>:/prefijoNuevo/cloud-up.svg</normaloff>:/prefijoNuevo/cloud-up.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="botonBorrar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Borrar</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #2C3E50</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../Recursos/img/bg.qrc">
         <normaloff>:/prefijoNuevo/close.svg</normaloff>:/prefijoNuevo/close.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="framedeTabla">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>911</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(38, 45, 67);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QTableWidget" name="tabla">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>891</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #2C3E50; /* Fondo oscuro */

}
    QHeaderView::section {
        background-color:  rgb(20, 24, 36);
    }</string>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellidos</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Num_Identificacion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha_Nacimiento</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Pais</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Teléfono</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E-mail</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sexo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Menores</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Recursos/img/bg.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
